policy/datasets.yaml

```diff
@@ -675,3 +675,7 @@ calexp_camera:  # Camera-level view of calexp after sky correction
     python: lsst.afw.image.ExposureF
     storage: FitsStorage
     level: None
+verify_job:  # Not persistable yet, but we can save a location for it.
```

Comment:
Perhaps you could add a comment that explains what it is?

Reply:
Is that ok?